### DISCUSSION TITLE
Start http server when using jstree v3.3.11 with external html data

### DISCUSSION THREAD

**Discussion by pte1601** [vakata/jstree#2491] [General]
Hello jstree users
I am trying to run jstree v3.3.11 with external html data (see attached file extract) and I am always getting the error: CORS request not http). Does this mean I need to start a development http server and load the external html data from there?
Thanks for your assistance
Greetings, Peter
[jstree.txt](https://github.com/vakata/jstree/files/5948916/jstree.txt)

**Comment by vakata** [comment]
Using the file protocol only works in Firefox - for all other browsers - use a dev http server. Unfortunately, this is a browser thing - nothing I can do on the jsTree side.

**Reply by pte1601** [reply]
Hi vacata,
thanks; 
I found out I also could disable CORS policy in firefox with setting config variable security.fileuri.strict_origin_policy to false. 
But using this 2nd solution the changed event does not load the data anymore

.on("changed.jstree", function (e, data) {
            var i,
              j,
              r = [];
            for (i = 0, j = data.selected.length; i < j; i++) {
              r.push(data.instance.get_node(data.selected[i]).a_attr.href);
            }
            $("#event_result").load(r[0]);
          });
What could be here the reason?
Thanks for your assistance
Greetings, Peter
